<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPageIntro">
   <property name="title">
    <string>Introduction</string>
   </property>
   <property name="subTitle">
    <string>Please read this page carefully!</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelIntro">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This wizard will guide you though the steps need to create a service integration plugin for MellowPlayer.&lt;/p&gt;&lt;p&gt;A service integration plugin let you add support for a new web-based streaming service.&lt;/p&gt;&lt;p&gt;A plugin is a directory which contains 4 files:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;description.html&lt;/span&gt;: this file contains the description of your plugin and will be visible in the service selection dialog.&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;integration.js&lt;/span&gt;: this file contains the code of your plugin. You just have to implement a few functions in javascript...&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;metadata.ini&lt;/span&gt;: this file contains some metadata about your plugin.&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;logo.svg&lt;/span&gt;: the logo of your plugin.&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;This wizard will take care of creating all those files...&lt;/p&gt;&lt;p&gt;See the &lt;a href=&quot;http://mellowplayer.readthedocs.org/en/latest/developers/plugins.html#functions-to-implement&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;API documentation&lt;/span&gt;&lt;/a&gt; for a description of each of these functions...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPageDetails">
   <property name="title">
    <string>Details</string>
   </property>
   <property name="subTitle">
    <string>Please fill in the below form about your plugin</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editAuthor"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Author website:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editAuthorUrl"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Servive name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editSvName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Service URL:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editSvUrl"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPageFinished">
   <property name="title">
    <string>Finished</string>
   </property>
   <property name="subTitle">
    <string>Your plugin has been created!</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your plugin has been created: &lt;a href=&quot;file://&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;%1&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Restart MellowPlayer and you plugin should appear in the list of available services.&lt;/p&gt;&lt;p&gt;It's now up to you to implement the body of the functions of &lt;span style=&quot; font-weight:600;&quot;&gt;integration.js&lt;/span&gt; and give a nice description of the service and your plugin in &lt;span style=&quot; font-weight:600;&quot;&gt;description.html.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Open plugin directory</string>
      </property>
      <property name="icon">
       <iconset theme="folder-open"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
